README.md

```diff
@@ -16,34 +16,38 @@ View the changelog on the [releases page](https://github.com/taskcluster/taskclu
 Requirements
 ------------
 
-This is tested on and should run on any of node `{5, 6, 7, 8}`.
+This is tested on and should run on any of node `{8, 10}`.
 
 Usage
 -----
 
 This library is used to manage schemas used within an application.
 Schemas are typically stored under `schemas/` in the repository root, with a directory per schema version.
 Schema files in the repository are `.yml` files, but will be published as `.json` files.
-The schema identifiers will be constructed based on the `rootUrl` supplied to the constructor.
 
 ```javascript
-let doc = {'what-is-this': 'it-is-the-json-you-wish-to-validate'};
+const SchemaSet = require('taskcluster-lib-validate');
 
-// Create a validator for you to use
-validate = await validator({
-  rootUrl: 'https://taskcluster.example.com',
+cosnt doc = {'what-is-this': 'it-is-the-json-you-wish-to-validate'};
+
+// Create a SchemaSet for you to use
+const schemaset = new SchemaSet({
   serviceName: 'someservice',
   constants: {'my-constant': 42},
 });
 
 // The loaded schemas are easily accessible
-console.log(validate.schemas)
+console.log(schemaset.schemas)
 // ↳ [{'id': 'first/schema', ...}, {'id': 'second/schema', ...}, ...]
 ```
 
 ### Validating
 
+You must get a `validator` out of a `SchemaSet` to use it.
+
 ```javascript
+
+const validate = schemaset.validator('https://some-taskcluster-root-url.com');
```

Comment:
worth mentioning that `validate` also has a `schemas` property, and that it contains id's and $ref's that are url's (meaning they can be fetched).